scripts/pylib/pytest-twister-harness/tests/helpers/shell_mcuboot_command_parser_test.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import textwrap
+
+from twister_harness.helpers.shell import ShellMCUbootCommandParsed, ShellMCUbootArea
+
+
+def test_if_mcuboot_command_output_is_parsed_two_areas() -> None:
+    cmd_output = textwrap.dedent("""
+        \x1b[1;32muart:~$ \x1b[mmcuboot
+        swap type: revert
+        confirmed: 0
+        primary area (1):
+        version: 0.0.2+0
+        image size: 68240
+        magic: good
+        swap type: test
+        copy done: set
+        image ok: unset
+        secondary area (3):
+        version: 0.0.0+0
+        image size: 68240
```

Comment:
This is not going to work, it will work right now with the zephyr commit you used when you made this but can guarantee by zephyr 3.5 this will not work because the output size has changed slightly

Reply:
It is only a unit test for `ShellMCUbootCommandParsed` helper class. When output of `mcuboot` command will change, it can be updated here, to test our helper class to be sure it still works as expected.
Some fields (like image size) are not checked here, there are added as an example of a real output.

Reply:
@nordicjm Just to be clear. It is a mock of an output here, not an expected output from a test.